Program_1.0.py

```diff
@@ -1,22 +1,74 @@
 import wikipedia
 import pprint
+from PIL import Image
+from textblob import TextBlob
+import glob
+import imgcompare
 from pymongo import MongoClient
 
 # Podlaczenie do lokalnej bazy danych
 client = MongoClient()
 db = client.countries
 
-def saving_country(country_input):
-    country = wikipedia.WikipediaPage(country_input).summary
+
+def saving_country_to_db(input_country):
+    country = wikipedia.WikipediaPage(input_country).summary
     country_data = {'name': country_name, 'Summary': country}
     db.countries.insert_one(country_data)
 
+
+def flag_name_from_path(path):
+    flag_name = ''
+    for letter in path[len(path)-5::-1]:
+        if letter != '/':
+            flag_name = flag_name + letter
+        else:
+            return flag_name[::-1]
+
+
+def comparing_flags(input_flag):
+    difference = 30.0
+    for file in glob.glob('Flags/*.png'):
+        test_flag = Image.open(file).resize(input_flag.size).convert('RGB')
+        percentage = imgcompare.image_diff_percent(test_flag, input_flag)
+        if percentage < difference:
```

Comment:
dalej źle 👎 ciągle może być fałszywy wynik

Reply:
dlaczego?

Reply:
No może być, ale i tak to mi zwraca najbardziej podobną flagę

Reply:
zwróci ci OSTATNIA flage ktorej percentage jest mniejszy niz 30 zamiast flagi ktora ma najmniejszy percetnage